app/utils/parse-vulnerable-api-finding.ts

```diff
@@ -54,27 +50,69 @@ function initializeVulnerableApiFinding(): VulnerableApiFinding {
       method: '',
       params: { key: '', token: '' },
       url: '',
+      cookies: {},
     },
     response: {
       headers: {},
       reason: '',
       status_code: 0,
       text: '',
       url: '',
+      cookies: {},
     },
     severity: '',
     confidence: '',
     description: '',
   };
 }
 
+function updateSection(
```

Comment:
can we move the function to its original location and have an updated ts doc like earlier